psh/test-touch-rootfs.py

```diff
@@ -0,0 +1,33 @@
+# Phoenix-RTOS
+#
+# phoenix-rtos-tests
+#
+# psh touch command test for rootfs targets
+#
+# Copyright 2022 Phoenix Systems
+# Author: Damian Loewnau
+#
+# This file is part of Phoenix-RTOS.
+#
+# %LICENSE%
+
+import psh.tools.psh as psh
+from psh.tools.common import assert_mtime_change
+
+
+def assert_hardlinks(p):
+    dates = {}
+    # all psh commands are hardlinks
+    psh_hardlinks = psh.get_commands(p)
+    for hardlink in psh_hardlinks:
```

Comment:
```suggestion
    for hardlink in psh.get_commands(p):
```
Maybe like this? We don't use hardlinks else where and it nicely says us that psh command is a hardlink.